code/game/machinery/vendors/generic_vendors.dm

```diff
@@ -738,12 +746,11 @@
 	armor = list(melee = 50, bullet = 20, laser = 20, energy = 20, bomb = 0, bio = 0, rad = 0, fire = 100, acid = 70)
 	resistance_flags = FIRE_PROOF
 	products = list(/obj/item/stack/cable_coil/random = 10, /obj/item/crowbar = 5, /obj/item/weldingtool = 3, /obj/item/wirecutters = 5,
-					/obj/item/wrench = 5, /obj/item/analyzer = 5, /obj/item/t_scanner = 5, /obj/item/screwdriver = 5)
+					/obj/item/wrench = 5, /obj/item/analyzer = 5, /obj/item/t_scanner = 5, /obj/item/screwdriver = 5, /obj/item/clothing/gloves/color/yellow = 1)
 	contraband = list(/obj/item/weldingtool/hugetank = 2, /obj/item/clothing/gloves/color/fyellow = 2)
-	premium = list(/obj/item/clothing/gloves/color/yellow = 1)
 	prices = list(/obj/item/stack/cable_coil/random = 20, /obj/item/crowbar = 75, /obj/item/weldingtool = 100,
 				/obj/item/wirecutters = 50, /obj/item/wrench = 75,/obj/item/analyzer = 25, /obj/item/t_scanner = 25,
-				/obj/item/screwdriver = 50, /obj/item/weldingtool/hugetank = 120)
+				/obj/item/screwdriver = 50, /obj/item/clothing/gloves/color/yellow = 250, /obj/item/weldingtool/hugetank = 120)
```

Comment:
it will be curious to see how insuls will be handled with the $250 price tag, looking forward to seeing if it will get bought every round or not 👀

Reply:
100% it will be bought every round